LayoutTests/imported/w3c/web-platform-tests/preload/link-header-modulepreload-expected.txt

```diff
@@ -0,0 +1,6 @@
+
+
+Harness Error (TIMEOUT), message = null
+
+TIMEOUT test that a header-preloaded module is loaded and consumed Test timed out
```

Comment:
It would be nice to skip tests that timeout to avoid slowing EWS, I know @cdumez did a bunch of work around that in november.

Reply:
Done.